benchmarks/nnls/solvers/baseline.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+
+from benchopt.base import BaseSolver
+
+
+class Solver(BaseSolver):
+    name = 'Baseline'  # proximal gradient, eventually accelerated
+
+    # any parameter defined here is accessible as a class attribute
+    parameters = {'use_acceleration': [False, True]}
+
+    def set_objective(self, X, y):
+        self.X, self.y = X, y
+
+    def run(self, n_iter):
+        L = np.linalg.norm(self.X, ord=2) ** 2
+
+        n_features = self.X.shape[1]
+        w = np.zeros(n_features)
+        t_new = 1
+        for _ in range(n_iter):
+            if self.use_acceleration:
+                t_old = t_new
+                t_new = (1 + np.sqrt(1 + 4 * t_old ** 2)) / 2
+                w_old = w.copy()
+            grad = self.X.T.dot(self.X.dot(w) - self.y)
+            w -= grad / L
+            w = np.clip(w, a_min=0, a_max=None)
+            if self.use_acceleration:
+                w += (t_old - 1.) / t_new * (w - w_old)
```

Comment:
@josephsalmon with this update rule I am not sure that this w is feasible. This is y_t in Beck and Teboulle notation, you need to return the iterate right after the prox has been applied (x_t, defined as w 2 lines above )

Reply:
this could explain why the check of feasibility is commented in objective

Reply:
the 2 issues are solved.

Reply:
You were so right. Sorry I copied that from @mathurinm  baseline in Lasso that needs to be corrected to.
Let me know if you are fine with the current version (actually w in the code is y_t, but the output value is correct now).
Might need some refactoring in a specific PR for the two FISTA implementation (nnls and lasso), wdyt?

Reply:
I don't know if you got my comment, doing it this way @josephsalmon your `w - w_old` is 0

Reply:
@tomMoral I am not sure I agree with what you wrote. Code is gone now but if you go to lasso.baseline, the code is correct to me. Before you enter the use_acceleration condition, w is x_t. Instead of the current line you could just write:
`w (y_t+1) = w (x_t) + ratio (w (x_t) - w_old (w_{t-1})`
but that would amount to the same, no ?